Himanshu_Kumar/project1-(calculator)/assets/js/cal_js.js

```diff
@@ -0,0 +1,138 @@
+const calculator = {
+  displayValue: '0',
+  firstOperand: null,
+  waitingForSecondOperand: false,
+  operator: null,
+};
+
+function fact(val){
+  let v=1;
+  for(let i=1; i<=val; i++){
+    v=v*i;
+  }
+  return v;
+}
+
+function inputDigit(digit) {
```

Comment:
This function might fail to display an expression properly before evaluating it

Reply:
i am not displaying expressions on the screen. The values calculated are working fine.